shared/jobs.lua

```diff
@@ -1,5 +1,7 @@
 QBShared = QBShared or {}
 QBShared.ForceJobDefaultDutyAtLogin = true -- true: Force duty state to jobdefaultDuty | false: set duty state from database last saved
+QBShared.QBJobsStatus = false -- true: qb-jobs resource is installed | false: qb-jobs resource is not installed
```

Comment:
This can be done dynamically with the following native:
https://docs.fivem.net/natives/?_0x4039B485

Reply:
But this gives the option to turn off qb-jobs for the built-in jobs and still use it for addon jobs.

Reply:
We can easily do a combination of the two